lib/styles/themes/base/def/colors.js

```diff
@@ -67,4 +75,7 @@ export default {
   TEXT_LIGHT: `
     color: var(--text-light);
   `,
+  BG_DEFAULT: `
```

Comment:
Wrong variable it should be more specific to what exactly does this affect in UI.